ffv1.md

```diff
@@ -152,25 +152,26 @@ a = b, a += b, a -= b, a *= b
 
 ### Pseudo-code
 
-Several components of FFV1 are described in this document using pseudo-code. Note that the pseudo-code is used for clarity in order to illustrate the structure of FFV1 and not intended to specify any particular implementation. The pseudo-code used is based upon the C programming language [@!ISO.9899.1990] as uses its `if/else`, `while` and `for` functions as well as functions defined within this document.
+FFV1 bitstream is described in this document using pseudo-code. Note that the pseudo-code is used for clarity in order to illustrate the structure of FFV1 and not intended to specify any particular implementation. The pseudo-code used is based upon the C programming language [@!ISO.9899.1990] as uses its `if/else`, `while` and `for` functions as well as functions defined within this document.
```

Comment:
prefer to start with `The FFV1 bitstream is`.

Reply:
Argh, my ugly English. Not the first time you do the remark if I remember well :).
Thanks, fixed.